app/controllers/vendors_controller.rb

```diff
@@ -0,0 +1,51 @@
+class VendorsController < ApplicationController
+  before_action :set_vendor, only: %i[show destroy]
```

Comment:
Should this controller authenticate users? (If not, we should ticket that.)

Reply:
FYI the authentication work was done simultaneously and got merged first.  I say ticket separately.

Reply:
https://github.com/sul-dlss/dataloading-management/issues/58